### PR TITLE
qh - issue #5

### LINKED ISSUE #5: Add Style
#### As a user I want to see a visually pleasing application.

__Story__:

Our web application is really starting taking shape. We have added a few
features that provide value to our customers, but some of our customers are
complaining that the website is a quite bland. It is time to spice it up by
applying a visually appealing, and easy to maintain user interface to our web
application.

__Acceptance criteria__:
- [x] The application has a header with navigation
- [x] Images are displayed in an aesthetically pleasing list of cards
- [ ] Form errors are shown inline with the related input field
- [x] I know how to use Bootstrap components like buttons and navbars in my
  application.
- [x] I know the very basics of CSS and SCSS
- [x] My SCSS files are properly named and organized using BEM convention
- [x] My app has a "sticky" footer.
- [x] My app has a few customized styles using bootstrap variable overrides.
- [x] My app is responsive and is usable on a phone.
- [x] The active link in the navigation header is differentiated.
- [x] I use a bootstrap "container" to center my app contents (not necessarily "fluid").

__Reference material__:
- https://getbootstrap.com/docs/4.1/getting-started/introduction/
- https://github.com/plataformatec/simple_form
- http://getbem.com/naming/

__Dependencies__:
- Image Index

### PR DESCRIPTION
**Pull request by zacalex**
Closes #5

### CODE REVIEW

File: test/controllers/images_controller_test.rb [hunk 1]
```diff
@@ -19,7 +19,7 @@ class ImagesControllerTest < ActionDispatch::IntegrationTest
 
     assert_response :ok
     assert_equal 'Invalid image URL / Error happends', flash[:danger]
-    assert_select '.js-title', 'Enter new image info'
+    # assert_select '.js-title', 'Enter new image info'
```
[the remark this answers is not in the document]
Comment (joshuasbates) [reply test/controllers/images_controller_test.rb]: I'd say, quickly address the "minor" issues (just be aware of the "super minor" ones -- no need to fix)

File: app/assets/stylesheets/application.scss [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  bottom: 0;
+  height: 50px;
+  width: 100%;
+  overflow: hidden;
+}
+
+.navbar-toggler-icon{
+  background-color: gray;
+}
+.my-form{
```
[the remark this answers is not in the document]
Comment (zacalex) [reply app/assets/stylesheets/application.scss]: fixed

File: app/views/layouts/application.html.erb [hunk 3]
```diff
@@ -11,19 +11,24 @@
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
 
     <!-- Bootstrap CSS -->
-    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta.2/css/bootstrap.min.css" integrity="sha384-PsH8R72JQ3SOdhVi3uxftmaW6Vc51MKb0q5P2rRUpPvrszuE4W1povHYgTpBfshb" crossorigin="anonymous">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css" integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO" crossorigin="anonymous">
+
+    <!--<link rel="stylesheet" type="text/css" href="../../assets/stylesheets/images/index.css">-->
```
[the remark this answers is not in the document]
Comment (zacalex) [reply app/views/layouts/application.html.erb]: removed

File: app/views/layouts/application.html.erb [hunk 4]
```diff
@@ -11,19 +11,24 @@
...
   </head>
   <body>
+
+    <%= render 'layouts/nav' %>
```
[the remark this answers is not in the document]
Comment (zacalex) [reply app/views/layouts/application.html.erb]: Thanks!

File: test/controllers/images_controller_test.rb (hunk 1 above)
[the remark this answers is not in the document]
Comment (zacalex) [reply test/controllers/images_controller_test.rb]: fixed, sorry for that, will notice next time, so that won't waste your time

**Pull request merged**